conda/core/portability.py

```diff
@@ -48,25 +48,34 @@ class _PaddingError(Exception):
     pass
 
 
+def _subdir_is_win(subdir: str) -> bool:
+    if "-" in subdir:
```

Comment:
Is that the only sign for a non-noarch subdir? I'm a little nervous diverging from `on_win` about this given that prefix replacement is a pretty central part of conda.

Reply:
Yes, it's the only sign